colibre/config.yml

```diff
@@ -148,11 +148,21 @@ scripts:
     output_file: gas_smoothing_lengths.png
     section: Histograms
     title: Gas Particle Smoothing Lengths
+  - filename: scripts/number_of_agn_thermal_injections.py
+    caption: A histogram showing the cumulative number of black holes with a given total number of thermal energy injections (less than or equal to the number of particles heated) the black hole has had throughout the simulation.
+    output_file: num_agn_thermal_injections.png
+    section: Black Holes
+    title: Cumulative number of AGN thermal injections
   - filename: scripts/max_temperatures.py
     caption: Maximal temperature recorded by gas particles throughout the entire simulation.
     output_file: gas_max_temperatures.png
-    section: Histograms
+    section: Maximal Temperatures
     title: Maximal Temperature reached by gas particles
+  - filename: scripts/max_temperature_redshift.py
+    caption: A diagram displaying maximal temperatures vs redshifts at which they were reached. The pixel colour indicates the number of stellar + gas particles in the pixel.
```

Comment:
Avoid 'a diagram' - this adds nothing. Also avoid `+` in free-text. Something like:

The maximal temperatures reached by all star and gas particles (density given by the colour map) against the redshift at which they were that temperature.

Reply:
Agreed!